main/simulator.cpp

```diff
@@ -1,9 +1,115 @@
 #include "main/simulator.h"
 
+Simulator::Simulator(float frequency, size_t numOfRaysPerRow, int collectors,
+                     float simulationRadius, std::string_view objPath,
+                     int depth, float sourcePower)
+    : frequency_(frequency), numOfRaysPerRow_(numOfRaysPerRow),
+      manager_(SceneManager(collectors, simulationRadius)), depth_(depth),
+      sourcePower_(sourcePower) {
+  // TODO: no implementation yet
+  // model_ = manager_.getModelTriangles();
+
+  if (!manager_.loadModel(objPath)) {
+    std::stringstream ss;
+    ss << "Invalid model path";
+    throw std::invalid_argument(ss.str());
```

Comment:
When I look at this, I noticed that I could just write std::invalid_argument("invalid model path"). There is no need to use std::stringstream here